components/priority/PriorityTable.vue

```diff
@@ -93,7 +124,16 @@ export default {
             else if (prioridade === 'Baixa') return 'yellow'
             else if (prioridade === 'Muito Baixa') return 'green'
         },
```

Comment:
Missing blank line between methods.

Reply:
Sorry, forgot to delete these components

Resolved.